openfoam-interface/internal-interface/wrapper/simphonyInterface.C

```diff
@@ -945,6 +943,9 @@ void foam_modifyNumerics(std::string name, std::string fvSch, std::string fvSol,
         cDIS()
     );
 
+    TimeMod* t = (TimeMod*)(runTimes[name]);
```

Comment:
@jmarcelogimenez  sounds like a good approach.  I am not criticizing it at all. 

> It is true that subclass TimeMod is never instantiated directly, but thanks to C++ polimorphism I can use its extended behavior.

I understand.  I just don't see where either `TimeMod` or a subclass of `TimeMod` is initialized.  Or is this a work-in-progress?